### ISSUE TITLE
build fails on add_ethernet() parameter tx_write_only

### ISSUE THREAD

**Issue by nitej** [sensille/litehm2#4]
It appears that the pull request that added the tx_write_only parameter for add_ethernet() was later updated to a new argument format specifying txslots_write_only. See enjoy-digital/litex@7fa7a4c72a994955637cf13ac760c32e0845604a

**Comment by sensille** [comment]
Yes, thanks, I'll fix that with my next push. I assume you already fixed that locally?

**Comment by nitej** [comment]
Yes, I updated tx_write_only = True to txslots_write_only=True on my local copy and it built the bitstream. I'm assuming the option for rxslots_write_only should be left at the default value of True, right?
